catalog/resource_sql_table.go

```diff
@@ -188,7 +194,25 @@ func (ti *SqlTableInfo) getOrCreateCluster(clusterName string, clustersAPI clust
 	return aclCluster.ClusterID, nil
 }
 
+func (ci *SqlColumnInfo) serializeIdentityColumnInfo() string {
+	var identityType string
+	switch ci.Identity {
+	case IdentityColumnAlways:
+		identityType = "BIGINT GENERATED ALWAYS AS IDENTITY"
+	case IdentityColumnDefault:
+		identityType = "BIGINT GENERATED BY DEFAULT AS IDENTITY"
+	}
+	return identityType
+}
+
 func (ti *SqlTableInfo) serializeColumnInfo(col SqlColumnInfo) string {
+	var colType string
```

Comment:
While testing this, I saw some cases where Identify was empty. This happens when SqlColumnInfo is build by this package (and not directly by TF). 
In such cases, the `colType` ends empty and will cause issues.
We can replace the `serializeIdentityColumnInfo` by a `getColumnType`

```
func (ci *SqlColumnInfo) getColumnType() string {
	var colType string
	switch ci.Identity {
	case IdentityColumnAlways:
		colType = "BIGINT GENERATED ALWAYS AS IDENTITY"
	case IdentityColumnDefault:
		colType = "BIGINT GENERATED BY DEFAULT AS IDENTITY"
        default:
                colType = col.Type
	}
	return colType

```

and then remove the if/else from here:
```
var colType = getColumnType()
```

Reply:
Done